src/niscope/metadata/niscope.proto

```diff
@@ -1439,6 +1439,7 @@ message FetchArrayMeasurementRequest {
     ArrayMeasurement array_meas_function = 4;
     sint32 array_meas_function_raw = 5;
   }
+  optional sint32 meas_wfm_size = 6;
```

Comment:
This is from https://github.com/ni/grpc-device/pull/844

https://github.com/ni/nimi-python/blob/master/generated/niscope/niscope/_grpc_stub_interpreter.py#L252 currently ignores the `measurement_waveform_size` parameter.